python/cudf/utils/utils.py

```diff
@@ -129,9 +129,6 @@ def standard_python_slice(len_idx, arg):
     return start, stop, step, slice_length
 
 
-list_types_tuple = (list, np.array)
```

Comment:
FWIW, I think that we should avoid indirections like this in the future.  I think that they hide details where they're needed, and obfuscate the code without much advantage.